docker-compose.prod.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+
+services:
+  rdf-differ:
+    container_name: rdf_differ
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    command: gunicorn --bind 0.0.0.0:${RDF_DIFFER_API_PORT} ${FLASK_APP}:app
```

Comment:
IMHO: this should be the last line in the docker file

`CMD ["gunicorn","--bind","0.0.0.0:${RDF_DIFFER_API_PORT}","${FLASK_APP}:app"]`